Update licenses.md

sanderrenes opened the pull request:
changed a typo and added a sentence on copyleft licences

rlanzafame commented:
Hey @sanderrenes thanks for the PR! I see your description says you "changed a typo and added a sentence on copyleft licenses" but I only see a change for the typo. Is it possible you forgot to commit/push the copyleft sentence?

Tom-van-Woudenberg commented:
> Hey @sanderrenes thanks for the PR! I see your description says you "changed a typo and added a sentence on copyleft licenses" but I only see a change for the typo. Is it possible you forgot to commit/push the copyleft sentence?

It's here https://github.com/TeachBooks/manual/pull/134/commits/682f6a9aefa3615e510b05d2fcfb630a59040f71#diff-cd18c593f689a7151a31440265d4eb3d47d30e99386b5c63d2b3fb8086496430R80

sanderrenes commented:
I added line 80 for clarification

Op vr 14 mrt 2025 06:50 schreef Tom van Woudenberg ***@***.***
>:

> Hey @sanderrenes <https://github.com/sanderrenes> thanks for the PR! I
> see your description says you "changed a typo and added a sentence on
> copyleft licenses" but I only see a change for the typo. Is it possible you
> forgot to commit/push the copyleft sentence?
>
> It's here 682f6a9
> #diff-cd18c593f689a7151a31440265d4eb3d47d30e99386b5c63d2b3fb8086496430R80
> <https://github.com/TeachBooks/manual/commit/682f6a9aefa3615e510b05d2fcfb630a59040f71#diff-cd18c593f689a7151a31440265d4eb3d47d30e99386b5c63d2b3fb8086496430R80>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TeachBooks/manual/pull/134#issuecomment-2723675930>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC7OJBJ2GBAPMZOPE7BSTT2UJU2ZAVCNFSM6AAAAABYYS5SEOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRTGY3TKOJTGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: Tom-van-Woudenberg]*Tom-van-Woudenberg* left a comment
> (TeachBooks/manual#134)
> <https://github.com/TeachBooks/manual/pull/134#issuecomment-2723675930>
>
> Hey @sanderrenes <https://github.com/sanderrenes> thanks for the PR! I
> see your description says you "changed a typo and added a sentence on
> copyleft licenses" but I only see a change for the typo. Is it possible you
> forgot to commit/push the copyleft sentence?
>
> It's here 682f6a9
> #diff-cd18c593f689a7151a31440265d4eb3d47d30e99386b5c63d2b3fb8086496430R80
> <https://github.com/TeachBooks/manual/commit/682f6a9aefa3615e510b05d2fcfb630a59040f71#diff-cd18c593f689a7151a31440265d4eb3d47d30e99386b5c63d2b3fb8086496430R80>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TeachBooks/manual/pull/134#issuecomment-2723675930>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC7OJBJ2GBAPMZOPE7BSTT2UJU2ZAVCNFSM6AAAAABYYS5SEOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRTGY3TKOJTGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

rlanzafame commented:
ah got it, thanks! it was hidden before for some reason.
